src/components/home/Home.tsx

```diff
@@ -1,28 +1,53 @@
 import "./Home.scss";
 
+import { AppDispatch, RootState } from "state/rootReducer";
 import { EXACT_TRANSACTION, LOGIN_PATH, SUBMISSIONS_PATH, TRANSACTIONS_PATH } from "routes/routes";
 import { Redirect, Route, RouteComponentProps, Switch, withRouter } from "react-router-dom";
+import { setLoggedIn, setLoggedOut, setLoggingIn } from "components/login/state/statusAction";
 
 import { Footer } from "components/footer/Footer";
 import Header from "components/header/Header";
 import { LoggedIn } from "components/logged-in/LoggedIn";
 import Login from "components/login/Login";
 import { LoginStatus } from "enums/loginStatus";
 import React from "react";
-import { RootState } from "state/rootReducer";
 import { SANDBOX_PATH } from "sandbox/Sandbox";
 import { connect } from "react-redux";
+import { handleAutoLogin } from "components/login/login-service/loginService";
 
 const mapStateToProps = (state: RootState) => ({
     logginStatus: state.loginState.status,

```

Comment:
I'm not sure how I missed this before
```suggestion
    loginStatus: state.loginState.status,

```